### ISSUE TITLE
Invalid Characters in CI

### ISSUE THREAD

**Issue by adenta** [presidentbeef/brakeman#1150]
### Background

Brakeman version: upgrading from 3.7.2 to 4.1.1
Rails version: 4.0.0
Ruby version: 2.4.3

### Issue
I am trying to update the brakeman gem, and keep getting weird issues about ASCII characters.

EG: `Invalid US-ASCII character "\xC2" | While compiling HAML in <<LOCATION>>`

Similar to [this](https://github.com/presidentbeef/brakeman/issues/102) issue, however; the solution provided does not fix our problems. This is happening in Buildkite, our CI enviroment, and not locally.

**Comment by presidentbeef** [comment]
Hm...if it's happening in CI and not locally, it sounds exactly like #102. Can you check what `LANG` is set to in your current environment and make sure CI matches it?

**Comment by adenta** [comment]
```
[1] dev(main)> ENV["LANG"]
=> "en_US.UTF-8"
[2] dev(main)> ENV["LC_ALL"]
=> nil
[3] dev(main)>
```
😕

**Comment by cmitz** [comment]
Hi @adenta and future readers, I came across this issue because I experienced the same. After a little digging, it seems that the problem lies with "Debian Jessie dropping the 'locales' dependency" affecting many systems including the official ruby Docker image. `en_US.UTF-8` is not available.

TL;DR: set the LANG to `C.UTF-8` in your CI environment (Docker, or whatever you use).

- More info: https://github.com/infosiftr/docker-library-docs/blob/master/ruby/content.md#encoding
- Even more info: https://github.com/docker-library/ruby/issues/70
- The Ruby dockerfile discussion: https://github.com/docker-library/ruby/issues/45

**Comment by presidentbeef** [comment]
Thanks @cmitz!

Did you happen to test with Brakeman 4.4.0? The default external encoding is now UTF-8, which should override any environment variables.

**Comment by cmitz** [comment]
Hi @presidentbeef, it looks like Brakeman 4.4.0 does not have this issue! 🚀

**Comment by presidentbeef** [comment]
@cmitz Awesome, thank you for testing!

Seems like I accidentally fixed this issue with #1288  :laughing:
